services/friendbot/internal/minion.go

```diff
@@ -138,11 +213,43 @@ func (minion *Minion) makeTx(destAddress string) ([32]byte, string, error) {
 	if err != nil {
 		return [32]byte{}, "", errors.Wrap(err, "unable to hash")
 	}
+	return txh, txe, err
+}
 
-	// Increment the in-memory sequence number, since the tx will be submitted.
-	_, err = minion.Account.IncrementSequenceNumber()
+func (minion *Minion) makePaymentTx(destAddress string) ([32]byte, string, error) {
+	createAccountOp := txnbuild.Payment{
```

Comment:
nit: typo? threw me off heh
```suggestion
	paymentOp := txnbuild.Payment{
```

Reply:
fc14d1bd